src/Build/Instance/RunningObjectTable.cs

```diff
@@ -50,10 +58,25 @@ public object GetObject(string itemName)
             return obj;
         }
 
-        private IMoniker CreateMoniker(string itemName)
+        private Task<IMoniker> CreateMoniker(string itemName)
```

Comment:
nit: This method should be called `CreateMonikerAsync` since it returns a `Task`, to conform to the convention.

Reply:
inlined